script.js

```diff
@@ -18,4 +18,336 @@ lists.forEach((list) => {
+
+  const seeLive = document.createElement('button');
+  seeLive.innerHTML = '<h2>See Live <img class="popup-links" src="./images/See-Live.png" alt="Live link"><h2>';
+  seeLive.className = 'see-more';
+  popupSeeMore.appendChild(seeLive);
+
+  const seeSource = document.createElement('button');
+  seeSource.className = 'see-more';
+  seeSource.innerHTML = '<h2>See Source <img class="popup-links" src="./images/github-white-icon.png" alt="github link"><h2>';
+  popupSeeMore.appendChild(seeSource);
+
+  popupClose.addEventListener('click', () => {
+    body.removeChild(projectPopup);
+    body.removeChild(overlay);
+  });
+});
+
+const projects = [
+  project2,
+  project3,
+  project4,
+  project1,
+  project5,
+  project6,
+];
+
+projects.className = 'projects';
+
+projects.forEach((project) => {
+  const projectdiv = document.createElement('div');
+  projectdiv.className = project.className;
+  projectsSection.appendChild(projectdiv);
+
+  const projTitle = document.createElement('div');
+  projTitle.className = 'professional';
+  projTitle.innerHTML = project.name;
+  projectdiv.appendChild(projTitle);
+
+  const projdesc = document.createElement('div');
+  projdesc.className = 'proj-desc';
+  projdesc.innerHTML = project.description;
+  projectdiv.appendChild(projdesc);
+
+  const projLangList = document.createElement('ul');
+  projLangList.className = 'buttons';
+  projectdiv.appendChild(projLangList);
+
+  const [tech1, tech2, tech3] = project.technologies;
+
+  const projLang1 = document.createElement('li');
+  projLang1.className = 'proj-lang';
+  projLang1.innerHTML = tech1;
+  projLangList.appendChild(projLang1);
+
+  const projLang2 = document.createElement('li');
+  projLang2.className = 'proj-lang';
+  projLang2.innerHTML = tech2;
+  projLangList.appendChild(projLang2);
+
+  const projLang3 = document.createElement('li');
+  projLang3.className = 'proj-lang';
+  projLang3.innerHTML = tech3;
+  projLangList.appendChild(projLang3);
+
+  const seeProject = document.createElement('button');
+  seeProject.innerHTML = '<h2>See Project<h2>';
+  seeProject.className = 'project-link-btn';
+  seeProject.innerHTML = '<h2>See Project<h2>';
+  projectdiv.appendChild(seeProject);
+  seeProject.addEventListener('click', () => {
+    const projectPopup = document.createElement('div');
+    projectPopup.className = 'project-popup';
+    body.appendChild(projectPopup);
+
+    const overlay = document.createElement('div');
+    overlay.id = 'overlay';
+    body.appendChild(overlay);
+
+    const popupTitle = document.createElement('div');
+    popupTitle.className = 'popup-title';
+    popupTitle.innerHTML = project.name;
+    projectPopup.appendChild(popupTitle);
+
+    const popupClose = document.createElement('img');
+    popupClose.className = 'popup-close';
+    popupClose.src = './images/close.png';
+    popupTitle.appendChild(popupClose);
+
+    const popupLangs = document.createElement('ul');
+    popupLangs.className = 'popup-buttons';
```

Comment:
- [ ] Kindly adjust the padding of buttons in order to match the figma design

Yours:
![image](https://user-images.githubusercontent.com/62705947/217575179-2a2aa62c-4a7b-4e35-86af-683c5fbfe889.png)

Design:
![image](https://user-images.githubusercontent.com/62705947/217575242-d1e165ae-3e1f-4c90-b4fd-04ef7a8c53b6.png)

Reply:
- [ ] Add paddings on the side and adjust the X button, check the images for more info

Yours:
![image](https://user-images.githubusercontent.com/62705947/217575917-81e89257-260c-451a-bb7a-efb336d2c0cd.png)

Design:
![image](https://user-images.githubusercontent.com/62705947/217576005-cfd98382-ceed-4219-b5a0-60ec888830ff.png)

Reply:
- Good job done so far, but as a result of the adjustments made following the last review, you seem to have tampered with your live buttons, please fix this 🙏. Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏

Your project:
<img width="812" alt="ergergaerg" src="https://user-images.githubusercontent.com/73607512/217650931-bf7f67a2-da60-4129-9d02-179098bd6ea1.PNG">

Figma design:
<img width="492" alt="q3wtgq34t" src="https://user-images.githubusercontent.com/73607512/217651019-3bd23e04-4524-4ab7-b218-13c304b4c20a.PNG">
- Also, as a result of the adjustments you made following the last review, you seem to have introduced a bug. Please fix this 🙏 
 Your project:
<img width="265" alt="q34tq34t" src="https://user-images.githubusercontent.com/73607512/217651518-a103d34e-a624-493f-a844-bcb7beb95904.PNG">

Design template:
<img width="147" alt="efQEFqe" src="https://user-images.githubusercontent.com/73607512/217651653-9e267d1e-0993-4d99-9377-4864952c383c.PNG">